elastic/apm-server
# Remove `aggregation.service_destinations.enabled`

## Motivation/summary

Removes the `apm-server.aggregation.service_destinations.enabled` config
setting. The APM Server will now aggregate service destination metrics
by default.

## Checklist




## How to test these changes

1. `docker-compose up -d`
2. `make`
3. `./apm-server -E apm-server.aggregation.service_destinations.enabled=false -E output.elasticsearch.username=admin -E output.elasticsearch.password=changeme -e`
4. Verify that the server contain this message `creating service destinations aggregation with config: {Interval:1m0s MaxGroups:10000}`.

## Related issues

Closes #6496
